guides/gatherers.md

```diff
@@ -381,3 +382,22 @@ facts:
 ```
 
 For extra information refer to [trento-project/agent/../gatherers/systemd_test.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/systemd_test.go)
+
+### verify_password
+
+**Argument required**: yes.
+
+This gatherer uses `getent` to determine whether a password from a certain user needs to be changed or not. It returns `true` if the password should be changed or `false` if not.
+
+Specification examples:
+```yaml
+facts:
+  - name: hacluster
```

Comment:
What about something like `name: hacluster_well_known_password_detected`? To me, it would better fit with what the gatherer is actually doing.

Reply:
I see your point. I don't exactly like the "well known" term as it sounds like something positive where it's not. I'll change it a bit but I get the point :+1: